src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -75,22 +79,33 @@ public async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, Htt
             }
         }
 
-        public string GetRandomString(int maxLength)
+        /// Gets a random ASCII string within specified length range
+        public string GetRandomString(int minLength, int maxLength, bool alphaNumericOnly = true)
         {
-            int length = _random.Next(0, maxLength);
+            int length = _random.Next(minLength, maxLength);
             var sb = new StringBuilder(length);
             for (int i = 0; i < length; i++)
             {
-                sb.Append((char)(_random.Next(0, 26) + 'a'));
+                if (alphaNumericOnly)
+                {
+                    // alpha character
+                    sb.Append(alphaNumeric[_random.Next(alphaNumeric.Length)]);
+                }
+                else
+                {
+                    // use a random ascii character
+                    sb.Append((char)_random.Next(0, 128));
```

Comment:
This will create invalid headers. Are we intending to make Kestrel or HttpClient error out?

Reply:
I'm url encoding in the context of headers, c.f. https://github.com/dotnet/corefx/pull/39774/files#diff-cc569911df914c81854b23cd80d8aa10R126